docs/USER_RECORD.md

```diff
@@ -226,6 +226,14 @@ This field is optional, when unset the user should not be considered part of any
 A user record with a realm set is never compatible (for the purpose of updates,
 see above) with a user record without one set, even if the `userName` field matches.
 
+`aliases` → An array of strings, each being a valid UNIX user name. If
+specified, a list of additional UNIX user names this record shall be known
+under. These are *alias* names only, the name in `userName` is always the
+primary name. Typically, a user record that carries this field shall be
+retreavable and resolvable under every name listed here, pretty much everywhere
+where the primary user name is allowed too. If logging in is attempted via an
```

Comment:
everywhere where -> everywhere ??

Reply:
"where" starts the relative clause here? I don't see the issue?

Reply:
Maybe, not sure, as you know I'm bad at English.

Reply:
Yeah. "Everywhere where" should just be "everywhere"

Also "allowed too" -> "allowed to". Or just drop it completely and end at the "is": "X should be resolvable everywhere Y is [resolvable]" vs "X should be resolvable everywhere Y is allowed to [be resolvable]"